ramendev/README.md

```
@@ -43,17 +41,16 @@ ramendev unconfig FILENAME
 
 ## Undeploying ramen on the hub and managed clusters
 
-Delete resources deployed by `ramendev deploy` on the hub and managed
-clusters.
+Delete resources deployed by `ramendev deploy` on the hub and managed clusters.
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)